modules/ROOT/pages/configure-a-test-organization.adoc

```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
 == Export and import an organization
 
 You can export an organization definition from Bonita Studio. The exported file is in XML, so can be modified. You can import an
-organization definition in this XML format that was exported from another Bonita Studio running the same version of Bonita, or that was constructed by some other method but has the same format. You can also import the organization file into Bonita Portal, ready for production.
+organization definition in this XML format that was exported from another Bonita Studio running the same version of Bonita, or that was constructed by some other method but has the same format. You can also import the organization file into Bonita Administrator Application, ready for production.
```

Comment:
in this XML format that was exported from another Bonita Studio => in the XML file that was exported from another Bonita Studio

Reply:
Added